packages/shared/lib/integrations/scripts/webhook/github-app-oauth-webhook-routing.ts

```diff
@@ -36,7 +36,7 @@ export default async function route(nango: Nango, integration: ProviderConfig, h
         await handleCreateWebhook(integration, body);
     }
 
-    await nango.executeScriptForWebhooks(integration, body, 'installation.id', 'installation_id');
+    return await nango.executeScriptForWebhooks(integration, body, 'installation.id', 'installation_id');
```

Comment:
it's really up to each and everyone here, on my side I prefer to always await ahah 
a rule that I feels works when reviewing coding style: if there is an eslint rule for that, then we pick a solution and no more debate; otherwise it still can be suggested but as personal opinion